tests/upgrade_path/test_upgrade_path.py

```diff
@@ -88,12 +89,16 @@ def prepare_ptf(ptfhost, duthost, tbinfo):
     ptfhost.shell("supervisorctl update")
 
 
-@pytest.fixture(scope="module")
-def ptf_params(duthosts, rand_one_dut_hostname, creds, tbinfo):
-    duthost = duthosts[rand_one_dut_hostname]
+def ptf_params(duthost, creds, tbinfo, upgrade_type):
 
+    reboot_command = get_reboot_command(duthost, upgrade_type)
     if duthost.facts['platform'] == 'x86_64-kvm_x86_64-r0':
-        reboot_limit_in_seconds = 150
+        reboot_limit_in_seconds = 200
```

Comment:
why do we need to increase reboot limit?

Reply:
This change was earlier made to `test_advanced_reboot`, however `test_upgrade_path` was not updated with the increased time allowance for KVM.

Now, I think it is a better idea to define a constant somewhere and let both reboot and upgrade tests use the same constant, instead of two definitions. I will add a change.

Reply:
Probably with another PR. this PR is already doing more than the title says.

Reply:
@vaibhavhd, @yxieca can you please provide HLD or some requirements why this have been changes
`elif 'warm-reboot' in self.rebootType:
                if isMellanoxDevice(self.duthost):
                    self.rebootLimit = 1
                else:
                    self.rebootLimit = 0`